arm-compute/2016-09-30/swagger/containerService.json

```diff
@@ -0,0 +1,609 @@
+              "description": "The ssh key configuration for Linux VMs."
+            }
+          },
+          "required": [
+            "adminUsername",
+            "ssh"
+          ],
+          "description": "Profile for Linux VMs."
+        },
+        "ContainerServiceSshConfiguration": {
+          "properties": {
+            "publicKeys": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ContainerServiceSshPublicKey"
+              },
+              "description": "The list of SSH public keys used to authenticate with Linux-based VMs."
+            }
+          },
+          "description": "SSH configuration for Linux-based VMs running on Azure.",
+          "required": ["publicKeys"]
+        },
+        "ContainerServiceSshPublicKey": {
+          "properties": {
+            "keyData": {
+              "type": "string",
+              "description": "Certificate public key used to authenticate with VMs through SSH. The certificate must be in PEM format with or without headers."
+            }
+          },
+          "required": [
+            "keyData"
+          ],
+          "description": "Contains information about SSH certificate public key data."
+        },
+        "ContainerServiceDiagnosticsProfile": {
+          "properties": {
+            "vmDiagnostics": {
+              "$ref": "#/definitions/ContainerServiceVMDiagnostics",
+              "description": "Profile for the container service VM diagnostic agent."
+            }
+          },
+          "required": ["vmDiagnostics"]
+        },
+        "ContainerServiceVMDiagnostics": {
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "description": "Whether the VM diagnostic agent should be provisioned on the VM."
+            },
+            "storageUri": {
+              "readOnly": true,
+              "type": "string",
+              "description": "The URI of the storage account where diagnostics are stored."
+            }
+          },
+          "description": "Describes VM diagnostics.",
+          "required": [
+            "enabled"
+          ]
+        },
+        "ContainerService": {
+          "properties": {
+            "properties": {
+              "x-ms-client-flatten": true,          
+              "$ref": "#/definitions/ContainerServiceProperties"
+            }
+          },
+          "allOf": [
+            {
+              "$ref": "#/definitions/Resource"
+            }
+          ],
+          "description": "Container service."
+        },
+        "ContainerServiceListResult": {
+          "properties": {
+            "value": {
+              "type": "array",
+              "items": {
+                 "$ref": "#/definitions/ContainerService"
+              },
+              "description": "The list of container services."
+            },
+            "nextLink": {
+              "value": {
```

Comment:
nextLink cannot be a complex object. It is a string. 
Example: `"nextLink": "https://management.azure.com/blahblah"`.
So the schema should look like this:
```
"nextLink": {
  "type": "string",
  "description": "the URL to get the next set of container service results."
}
```